recipes/rb-escape_utils/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "escape_utils" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+    sha256: dffb7010922880ace6ceed642156c64e2a64620f27e0849f43bc4f68fd3c2c09
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+    - cp {{ name }}-*/LICENSE .
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+  host:
+    - ruby <2.6
+  run:
+    - ruby <2.6
+
+test:
+  commands:
+    - ruby -r {{ name }} -e 'exit 0'
+
+about:
+  home: https://rubygems.org/gems/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Faster string escaping routines for your ruby apps
+  doc_url: https://www.rubydoc.info/gems/{{ name }}
+  dev_url: https://github.com/brianmario/escape_utils
+
+extra:
+  recipe-maintainers:
+    - svengiegerich
```

happy to co-maintain this. :)

```suggestion
    - svengiegerich
    - janjagusch
```